package-lock.json

```diff
@@ -3610,9 +3610,9 @@
       "dev": true
     },
     "node_modules/@jellyfin/sdk": {
-      "version": "0.0.0-unstable.202307130502",
-      "resolved": "https://registry.npmjs.org/@jellyfin/sdk/-/sdk-0.0.0-unstable.202307130502.tgz",
-      "integrity": "sha512-1+GXATaJLP5akFnUrpxYzoshLtTPZXJEdy/ozhY1g/DkULlz4LthLTaTJ2qImF0mb8Ayk7LNbh00n4ATk0JycA==",
+      "version": "0.0.0-unstable.202312060501",
```

Comment:
This probably needs updated again to have the new (old) values. :upside_down_face:

Reply:
Ah yes that would probably help, I'm away from PC ATM but I'll update values tomorrow 😀

Reply:
Should be updated now :+1: